src/main/java/com/contrastsecurity/sdk/ContrastSDK.java

```diff
@@ -1830,13 +1831,29 @@ public HttpURLConnection makeConnection(String url, String method) throws IOExce
       connection.setRequestProperty(RequestConstants.TELEMETRY_INTEGRATION_VERSION, version);
```

Comment:
Let's remove these proprietary headers. The User-Agent is intended to replace these headers.